udata_gouvfr/templates/nec_mergitur.html

```diff
@@ -0,0 +1,85 @@
+{% extends theme("base.html") %}
+
+{% set meta = {
+    'title': 'Nec Mergitur',
+    'description': "Un Hackathon pour renforcer la sécurité dans Paris",
+    'keywords': [
+        'necmergitur',
+        'paris',
+        'fluctuat',
+        'nec',
+        'mergitur',
+        'hackathon',
+        'évènement',
```

Comment:
@vinyll : I whish I've known that 4 years ago, when someone bothered me about it !!!